08-projects.Rmd

```diff
@@ -6607,39 +6607,37 @@ F.R.I.E.N.D.S is a popular tv show. They have an interesting naming convention f
 - First 5 results of the query.
 ```
 
-The next couple of questions should be complete in the same language. You can use either R or Python, but you must use the same for both questions.
+##### 4. Now that you have a working query, connect to the database and run the query to get the data into an R data frame. In previous projects, we learned how to used regular expressions to search for text. For each character, how many episodes `primary_title`s contained their name? 
 
-##### 4. Now that you have a working query, connect to the database and run the query to get the data into an R or `pandas` data frame. In previous projects, we learned how to used regular expressions to search for text. For each character, how many episodes `primary_title`s contained their name? 
-
-**Relevant topics:** [SQL in R](#r-sql), [SQL in Python](#sql-in-python), [grep](#r-grep)
+**Relevant topics:** [SQL in R](#r-sql), [grep](#r-grep)
 
 ```{block, type="bbox"}
 **Item(s) to submit:**
 
-- R or Python code in a code chunk that was used to find the solution.
+- R code in a code chunk that was used to find the solution.
 - The solution pasted below the code chunk.
 ```
 
-##### 5. Create a graphic showing our results in (2) using your favorite package. Make sure the plot has a good title, x-label, y-label, and try to incorporate some of the following colors: #273c8b, #bd253a, #016f7c, #f56934, #016c5a, #9055b1, #eaab37.
+##### 5. Create a graphic showing our results in (4) using your favorite package. Make sure the plot has a good title, x-label, y-label, and try to incorporate some of the following colors: #273c8b, #bd253a, #016f7c, #f56934, #016c5a, #9055b1, #eaab37.
```

Comment:
Strictly speaking, there is no problem for the students to make a plot for Question 2 (which is counting the types in 2017). But according to [the solution folder](https://github.com/TheDataMine/fall2020-stat29000/tree/master/project15), Question 4 (counting names) seems to be intended initially.

Reply:
Yep